src/runtime/server/services/nuxtAuthHandler.ts

```diff
@@ -18,6 +18,11 @@ let preparedAuthHandler: ReturnType<typeof eventHandler> | undefined
 let usedSecret: string | undefined
 const SUPPORTED_ACTIONS: NextAuthAction[] = ['providers', 'session', 'csrf', 'signin', 'signout', 'callback', 'verify-request', 'error', '_log']
 
+export const ERROR_MESSAGES = {
```

Comment:
One feedback I got from the streamer using `nuxt-auth` for the first time was that our error messages were pretty long and spammed the console. 

Do we maybe want to shorten them here and have a dedicated part of our docs list fixes for each error? Then we can link that page in the error message and go more into detail into what the issue is?

Reply:
Super idea! I've added it (: